vpr/src/route/netlist_routers.h

```diff
@@ -53,6 +56,9 @@ class NetlistRouter {
      * \return RouteIterResults for this iteration. */
     virtual RouteIterResults route_netlist(int itry, float pres_fac, float worst_neg_slack) = 0;
 
+    /** Handle net bounding box updates. No-op for the serial router */
```

Comment:
Explain how you handle them (pass to the partition tree class to move the net up or down in the partition tree based on its new bounding box).